README.md

```diff
@@ -1,9 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+--- 
 
+This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
 
+---
 
---- 
+Follow these instructions if you wish to clone the repo:
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+## 01 Project Setup
+
+1.  Install dotenv dependency with command: `npm install dotenv`
+2.  Create .env files in the main directory:
+    - `.env.development` for development database
+    - `.env.test` for test database
+3.  Create environment variables in respective files:
+    - `PGDATABASE=development_database_name`
+    - `PGDATABASE=test_database_name`
```

Comment:
I'd be a bit more specific about these database names, they HAVE to be the same as they are in the setup.sql file or the server won't work :)